swagger/src/main/scala/org/http4s/rho/swagger/TypeBuilder.scala

```diff
@@ -80,6 +80,35 @@ object TypeBuilder {
 
           models ++ generics ++ children
 
+        case tpe@TypeRef(_, sym: Symbol, tpeArgs: List[Type]) if isSumType(sym) =>
+          // TODO promote methods on sealed trait from children to model
+          modelToSwagger(tpe, sfs).map({ m =>
```

Comment:
I'm generally not one to complain about styel, but can we remove the extra `(` in `.map({ m =>`? Same below on the `.flatMap` below. If you break this down into named methods the `flatMap` should disappear.